src/sections/About.jsx

```diff
@@ -20,9 +25,85 @@ const AboutContainer = styled.div`
     // aspect-ratio: 428 / 1344;
   }
 `
+
+const AboutBg = styled.img`
+  position: absolute;
+  top: 0;
+  width: 100%;
+  height: auto;
+  // Makes background non-selectable
+  -moz-user-select: none;
+  -webkit-user-select: none;
+  user-select: none;
+`
+
+const AboutTitle = styled.h2`
+  font-family: 'HK Grotesk';
+  font-weight: bold;
+  text-align: center;
+  color: #4D4B4F;
+  z-index: 1;
+  width: 100vw;
+  display: block;
+  position: relative;
+  top: 15%;
+  font-size: 3em;
+  width: 80%;
+  text-align: center;
+`
+
+const AboutDescription = styled.p`
+  font-family: 'HK Grotesk';
+  top: 5px;
+  font-size: 1em;
+  color: #4D4B4F;
+  z-index: 1;
+  position: relative;
+  width: 50%;
+  left: 10%;
+  margin-top: 20%;
+`
+
+const swingSign = keyframes`
+ from { transform: rotate(3deg); }
+ to { transform: rotate(-3deg); }
+
+`
+
+const OpenSignImage = styled.img`
+  position: absolute;
+  right: 6.5%;
+  top: 56vh;
+  animation-name: ${swingSign};
+  animation-duration: 2s;
+  animation-iteration-count: infinite;
+  animation-direction: alternate;
+  transform-origin: 0px 50%;
+`
+
 const About = () => (
   <AboutContainer>
```

Comment:
Text position looks slightly off (I'm viewing in a MBP viewport)

<img width="779" alt="image" src="https://user-images.githubusercontent.com/26179641/217330136-fc3b9c4a-81b3-4285-96c8-984f90a4d621.png">